app/main.py

```diff
@@ -1 +1,68 @@
-#  write your code here
+class Animal:
+    list_animals = []
```

Comment:
There is no need to indicate type in the attribute name, remove it
```suggestion
    animals = []
```